<launch>
    <arg name="subject" />
    <node pkg="hrl_head_tracking" name="r_head_tracking_srv" type="head_tracking_srv"
          output="screen" args="$(find hrl_head_tracking)/data/$(arg subject)_r_face.bag">
        <param name="max_iters" value="300" />
        <param name="color_weight" value="0.0" />
        <param name="icp_max_corresp" value="0.3" />
        <param name="icp_trans_eps" value="0.0000000001" />
        <remap from="/head_registration" to="/head_registration_r" />
    </node>
    <node pkg="hrl_head_tracking" name="l_head_tracking_srv" type="head_tracking_srv"
          output="screen" args="$(find hrl_head_tracking)/data/$(arg subject)_l_face.bag">
        <param name="max_iters" value="300" />
        <param name="color_weight" value="0.0" />
        <param name="icp_max_corresp" value="0.3" />
        <param name="icp_trans_eps" value="0.0000000001" />
        <remap from="/head_registration" to="/head_registration_l" />
    </node>
    <node pkg="hrl_face_adls" name="registration_loader" type="registration_loader.py"
          output="screen" args="$(find hrl_ellipsoidal_control)/data/$(arg subject)_r_ellipsoid_registration.bag $(find hrl_ellipsoidal_control)/data/$(arg subject)_l_ellipsoid_registration.bag" >
        <remap from="/feedback" to="/wt_log_out" />
    </node>
    <node pkg="hrl_head_tracking" name="head_alignment_confirm" type="head_alignment_confirm"
          output="screen" />
</launch>
